packages/livekit-rtc/src/audio_source.ts

```diff
@@ -14,14 +13,14 @@ import type {
 } from './proto/audio_frame_pb.js';
 import {
   AudioSourceType,
-  CaptureAudioFrameRequestSchema,
-  ClearAudioBufferRequestSchema,
-  NewAudioSourceRequestSchema,
+  CaptureAudioFrameRequest,
+  ClearAudioBufferRequest,
+  NewAudioSourceRequest,
 } from './proto/audio_frame_pb.js';
 
 export class AudioSource {
   /** @internal */
-  info: AudioSourceInfo;
+  info?: AudioSourceInfo;
```

Comment:
in what situation would this be missing?

Reply:
unsure, but in general i prefer to use `?` over `!` if the former works. could be swayed to replace it with an exclamation point later in the file

Reply:
if we expect an `AudioSource` to always have a `AudioSourceInfo` (which I think we do as it's required in the FFI protocol), I think we should reflect that on the user facing API.